cmd/jag/commands/scan.go

```diff
@@ -174,7 +178,7 @@ func scanAndPickDevice(ctx context.Context, scanTimeout time.Duration, port uint
 	}
 
 	if len(devices) == 0 {
-		return nil, false, fmt.Errorf("didn't find any Jaguar devices")
+		return nil, false, fmt.Errorf("didn't find any Jaguar devices.\nPerhaps you need to be on the same wifi as the device.\nYou can also specify the IP address of the device.")
```

Comment:
Not sure it's common to have error objects with multiple lines, but Go and errors...